packages/docs-ui/README.md

```diff
@@ -0,0 +1,39 @@
+<img src="https://raw.githubusercontent.com/seek-oss/braid-design-system/HEAD/packages/docs-ui/images/braid-docs-ui-logo.svg#gh-light-mode-only" alt="Braid Docs UI" title="Braid Docs UI" width="80px">
+<img src="https://raw.githubusercontent.com/seek-oss/braid-design-system/HEAD/packages/docs-ui/images/braid-docs-ui-logo-inverted.svg#gh-dark-mode-only" alt="Braid Docs UI" title="Braid Docs UI" width="80px" />
+<br/>
+
+# docs-ui
+
+Components for building documentation sites with a consistent user experience to the [Braid Design System] website.
+
+```bash
+npm install @braid-design-system/docs-ui
+```
+
+- [LinkableHeading](#linkableheading)
+
+## Usage
+
+### `LinkableHeading`
+
+A heading wrapped in a link to a hash that is generated based on the heading content, enabling deep links within documentation pages.
+
+#### Example
+
+```tsx
+import { LinkableHeading } from '@braid-design-system/docs-ui';
+
+<LinkableHeading>Section heading</LinkableHeading>;
+```
+
+#### Props
+
+| props     | value            | description                                                                                                                                                               |
+| --------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| level     | [`HeadingLevel`] | The heading size (defaults to `3`)                                                                                                                                        |
+| component | `string`         | HTML element to render, e.g. &ldquo;h1&rdquo;, &ldquo;h2&rdquo;, etc.                                                                                                     |
+| children  | `string`         | Visible heading content, converted to slug (e.g. &ldquo;section-heading&rdquo;) to be used as hash link.<br/>Note: If the heading                                         |
```

Comment:
If the heading what?

Reply:
Oh whoops, will add the fix in the follow up PR